exporters/auto/build.gradle

```diff
@@ -41,12 +42,24 @@ dependencies {
 	}
 }
 
-
 shadowJar {
+	// gradle warns since explicit or implicit dependency on this shadowJar is not declared
+	dependsOn ':shared-resourcemapping:shadowJar'
```

Comment:
Does this cause the resource mapping to be shadowed twice?

Reply:
If you mean there are duplicate classes in the resulting JAR, then no. 

If you mean the task ":shared-resourcemapping:shadowJar" is executed twice - 
 - `./gradlew :exporter-auto:shadowJar` - results in 1 invocation of the mentioned task. 
 - `./gradlew shadowJar` - also results in a single invocation of the mentioned task. 
 
#### Some additional info - 
In this current version of gradle (v8), without explicitly declaring a dependency on this task (:shared-resourcemapping:shadowJar), when building the shadowJar for exporter-auto, the executed task is `:shared-resourcemapping:jar` - this only brings in the source classes from the resourcemapping module and not the transitive dependencies in the exporter-auto JAR. 
 
 Not sure what was the prior behavior in gradle v7.